src/language.rs

```diff
@@ -57,6 +57,20 @@ impl Language {
     }
 }
 
+/// Reads an `ll_CC` formatted string into a [`Language`].
```

Comment:
This should probably be a normal comment (not a doc comment).  It also took me a bit to understand what `ll_CC` meant (full format would also be "`lang_COUNTRY.Encoding`, with `lang` being a 2-letter language code, and `COUNTRY` being 2-letter country code").